docker-compose.yml

```diff
@@ -4,14 +4,20 @@ networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:${IMAGE_VERSION:-2.0.0}
```

Comment:
The `image` field uses a variable `IMAGE_VERSION` with a default value of `2.0.0`. Ensure that this variable is defined in your environment or `.env` file if you intend to override the default version.